helper.js

```diff
@@ -1,25 +1,42 @@
 const fs = require("fs-extra");
 const path = require("path");
 
-const dataDirPath = path.resolve(__dirname, "./temp/data");
-const attachDirPath = path.resolve(__dirname, "./temp/images");
+let dataDirPath = '';
+let attachDirPath = '';
+let tempDirPath = path.resolve(__dirname, "./temp");
+
+async function setUpPaths(pathToDirectory) {
+  if (pathToDirectory !== undefined) {
+    tempDirPath = path.resolve(pathToDirectory, "./temp");
+  }
+
+  dataDirPath = path.resolve(tempDirPath, "./data");
+  attachDirPath = path.resolve(tempDirPath, "./images");
+
+  await fs.remove(tempDirPath);
+  await fs.mkdirs(dataDirPath);
+  await fs.mkdirs(attachDirPath);
+};
 
 const distDirName = `./jest-html-reporters-attach`;
 
 /**
  *
  * @param {Buffer | string} attach
  * @param {string} description
+ * @param {object} context. Optional. It contains custom configs
  */
 const addAttach = async (attach, description, context) => {
-  const { testPath, testName } = getJestGlobalData(context);
+  await setUpPaths(context.tempDirPath);
```

Comment:
If the `setUpPaths` function removes the `tempDir` totally every time, we can only get one piece of data finally.

Reply:
thank you, I'll think about how to rewrite this point